internal/streams/stream_provider_server_side.go

```diff
@@ -109,6 +110,7 @@ func (r *serverSideEnvStreamRepository) Replay(channel, id string) chan eventsou
 // getReplayEvent will return a ServerSidePutEvent with all the data needed for a Replay.
```

Comment:
This function is only called when we have a new or reconnecting client.
So some small delay here should be acceptable.